tracer/src/Datadog.Trace/Telemetry/DTOs/LogMessageData.cs

```diff
@@ -0,0 +1,21 @@
+// <copyright file="LogMessageData.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Vendors.Newtonsoft.Json.Converters;
```

Comment:
nit:
```suggestion
using Datadog.Trace.Vendors.Newtonsoft.Json.Converters;

#nullable enable
```

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/commit/7ba4bb6bbbcb9992e7874a4625f5a783bbf05002.